lib/mls_vectors/test/fips.h

```diff
@@ -0,0 +1,7 @@
+#include <mls/crypto.h>
+
+bool
+fips();
+
+bool
+is_fips_approved(mls::CipherSuite::ID id);
```

Comment:
These methods seem duplicative of `fips` and `fips_disable`?

Reply:
This method is no longer needed, so removed in the current commit.